backend/config/packages/agi_gitlab.yaml

```diff
@@ -0,0 +1,14 @@
+framework:
+    messenger:
+        routing:
+            App\Github\Message\Async\*: async
```

Comment:
BOT4:
The routing configuration is incorrectly targeting `Github` entities (`App\Github\Message\Async\*`) instead of `Gitlab`. This needs to be corrected to match the intended GitLab functionality.